UserInterface/MenuItemDemos/MenuItemDemos/MenuItemDemos/MenuItemCodePage.cs

```diff
@@ -0,0 +1,79 @@
+﻿using MenuItemDemos.Services;
+using System;
+
+using Xamarin.Forms;
+
+namespace MenuItemDemos
+{
+    public class MenuItemCodePage : ContentPage
+    {
+        Label messageLabel;
+
+        public MenuItemCodePage()
+        {
+            Padding = 10;
+            Title = "MenuItem Code Demo";
+
+            messageLabel = new Label
+            {
+                TextColor = Color.Red,
+                HorizontalOptions = LayoutOptions.CenterAndExpand,
+                VerticalOptions = LayoutOptions.Start
+            };
+
+            DataTemplate dataTemplate = new DataTemplate(() =>
+            {
+                Label label = new Label();
```

Comment:
Think you should add another Label somewhere telling the user what to do to reveal the context menu.